test/http/http.mli

```diff
@@ -0,0 +1,15 @@
+(** Http helpers used for tests *)
+
+module Server : sig
+  (* Dummy http server that accepts a single request *)
```

Comment:
It's really up to the client whether the server accepts a single request. Since this module exposes `accept` then a client may implement a more sophisticated server by calling `accept` multiple times.

Reply:
Revised.